routes/users.js

```diff
@@ -0,0 +1,127 @@
+// Routes relating to accounts here
+const express = require('express');
+
+const userRouter = express();
+const admin = require('../firebase');
+const { db } = require('../config');
+
+userRouter.use(express.json());
+
+const SQLQueries = {
+  GetAllUsers: `SELECT * FROM "Users"`,
+  GetSpecificUserByUserId: 'SELECT * FROM "Users" WHERE user_id = $1',
+  GetSpecificUserByEmail: 'SELECT * FROM "Users" WHERE email = $1',
+  DeleteUser: 'DELETE FROM "Users" WHERE user_id = $1',
+  CreateUser: 'INSERT INTO "Users" (first_name, last_name, email, user_id) VALUES ($1, $2, $3, $4)',
+  UpdateUser: 'UPDATE "Users" SET first_name = $1, last_name = $2 WHERE user_id = $3',
+  Return: ' RETURNING *',
+};
```

Comment:
Would it be possible to move the queries to a user services file instead?

Reply:
moved